EventFilter/HcalRawToDigi/plugins/PackerHelp.h

```diff
@@ -549,7 +549,11 @@ class UHTRpacker{
     // loop over words in dataframe
     for( int iTS = 0 ; iTS < qiedf->size() ; iTS++ ){
       // push data into uhtr data container
-      uhtrs[uhtrIndex].push_back(qiedf->sample(iTS).raw());
+      auto raw = qiedf->sample(iTS).raw();
+      // Add SOI information
+      if (iTS == qiedf->presamples())
+         raw |= 0x4000;
```

Comment:
better to define a mask with this value as a const in the `TPHeaderSpec` namespace